modules/stats.php

```diff
@@ -545,21 +724,27 @@ function stats_configuration_load() {
 			if ( isset( $_POST["role_$role"] ) && $_POST["role_$role"] )
 				$options['roles'][] = $role;
 
-		$options['count_roles'] = array();
+			$options['count_roles'] = array();
```

Comment:
This line doesn't belong to the previous `if`, nor to the `foreach`,  it shouldn't be indented.